### DISCUSSION TITLE
How to contribute to the repo?

### DISCUSSION THREAD

**Discussion by irdcat** [opwvhk/avro-schema-support#113] [Q&A] [answered]
Hi,

According to Contrubuting.md pull requests are welcome, however when I try to push something from my branch, request gets rejected with following message:

```
ERROR: Permission to opwvhk/avro-schema-support.git denied to irdcat.
fatal: Could not read from remote repository.      

Please make sure you have the correct access rights
and the repository exists.
```

Thanks for help in advance.

**Comment by opwvhk** [comment, marked as answer]
The error "fatal: Could not read from remote repository." is strange.

What should work:
1. Fork the repository
2. Make your changes in your fork (using a new branch is advisable)
3. Open a PR to merge your changes in the target repo

Does this work?

**Reply by irdcat** [reply]
It worked. Thank you

**Reply by opwvhk** [reply]
Thank you for verifying. I'll update the contribution guide.
